src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -76,11 +82,17 @@ public void onSwitchTo() {
 	@FXML
 	public void onPause(ActionEvent event) {
 		if (pf.isRunning()) {
-			pf.stopGame();
+			try {
+				pf.stopGameAndWait();
+			} catch (InterruptedException ex) { }
```

Comment:
There is no error displayed for me.
On top of that, any error that would be thrown there, should be silently ignored, and will not impact the game. It only means that the game does not pause directly, but that it will pause eventually.

<layout